CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ To add a new rule, we suggest following these steps:
 9. Add the rule to the vale/styles/jargonLint or vale/styles/jargonExtra directory.
     - To jargonLint, if the rule is generally applicable.
     - To jargonExtra, if the rule is reserved for specific situations.
-10. Add your test files to the "tests" folder in whichever directory you selected in the earlier step.
+10. Add your test files to the vale/tests directory.
```

Comment:
I'm not even sure it has to be in vale/tests. It could be in tests/ folder at the root, but I'm fine with this change